test/e2e/schedulingbase/volume_binding.go

```diff
@@ -363,6 +366,39 @@ var _ = ginkgo.Describe("Volume Binding Test", func() {
 			validateStatefulSet(ctx, config, ss, false)
 		})
 	})
+
+	f.Context("Dynamic Provisioning", f.WithSlow(), func() {
```

Comment:
Is this a dedicated case just for volcano?

Reply:
yes, since we do not have a cloud vendor offering PVs, I use kind's "standard" storageclass to provide PVs